app/controllers/arf_reports_controller.rb

```diff
@@ -45,6 +45,21 @@ def destroy
     end
   end
 
+  def expire_reports
+  end
+
+  def submit_expire_reports
+    days = Integer(params[:data][:days]) rescue nil
+    if days
+      ForemanOpenscap::ArfReport.expire :timerange => days.days
```

Comment:
won't this take a long time? we initially had an expire opt in the UI, and converted it into a rake / cron task instead.